paasta_tools/contrib/emit_allocated_cpu_metrics.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+import logging
+
+import yelp_meteorite
+
+from paasta_tools.cli.utils import get_instance_config
+from paasta_tools.utils import get_services_for_cluster
+from paasta_tools.utils import load_system_paasta_config
+
+log = logging.getLogger(__name__)
+
+
+def emit_metrics_for_type(instance_type):
+    cluster = load_system_paasta_config().get_cluster()
+    instances = get_services_for_cluster(
+        cluster=cluster,
+        instance_type=instance_type,
+    )
+
+    for service, instance in instances:
+        service_instance_config = get_instance_config(
+            service=service,
+            instance=instance,
+            cluster=cluster,
+        )
+        cpus = service_instance_config.get_cpus()
+        dimensions = {
+            'service_name': service_instance_config.service,
+            'paasta_cluster': service_instance_config.cluster,
+            'instance_name': service_instance_config.instance,
+        }
+
+        log.info("Emitting paasta.service.cpus = {} with dimensions {}".format(
+            cpus, dimensions,
+        ))
+        gauge = yelp_meteorite.create_gauge('paasta.service.cpus', dimensions)
```

Comment:
While we're here, can we add .mem, .disk, and maybe .instance_count? (I think we're already doing instance count from autoscaled services, but it would feel more complete to have them all.)

Having mem and disk and such would allow us to make better dashboards along the lines of the will-yelp-main-fit dashboard. Instance count would allow us to calculate total cost of services.

Reply:
...maybe also min_instances and max_instances?